apps/app/src/configs/fulfillment.ts

```diff
@@ -1,40 +1,115 @@
-export const fulfillmentConfig = {
-  default: {
-    sendOrder: true,
-    pickUp: true,
-    mailOrder: false,
-    mailOrderProviders: [],
-    sendToPatient: true,
-    sendToPatientUsers: [] // its weird to have this here, but its temporary and keeps ts from complaining
-  },
-  // Weekend Health
-  [process.env.REACT_APP_WEEKEND_ORG_ID as string]: {
-    sendOrder: false,
-    pickUp: true,
-    mailOrder: false,
-    mailOrderProviders: [],
-    sendToPatient: false,
-    sendToPatientUsers: []
+export enum Envs {
+  boson = 'boson',
+  neutron = 'neutron',
+  photon = 'photon'
+}
+type FulfillmentConfig = {
+  [key in Envs]: {
+    [key: string]: {
+      sendOrder: boolean;
+      pickUp: boolean;
+      mailOrder: boolean;
+      mailOrderProviders: string[];
+      sendToPatient: boolean;
+      sendToPatientUsers: string[];
+    };
+  };
+};
+
+export const fulfillmentConfig: FulfillmentConfig = {
+  boson: {
```

Comment:
Could we maybe do different files? `fulfillment.boson.ts` or s/e

Reply:
sure!